doc/api/deprecations.md

```diff
@@ -2514,6 +2514,22 @@ Type: Documentation-only
 Prefer [`response.socket`][] over [`response.connection`] and
 [`request.socket`][] over [`request.connection`].
 
+<a id="DEP0XXX"></a>
+### DEP0XXX: http finished
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/28679
+    description: Documentation-only deprecation.
+-->
+
+Type: Documentation-only
+
+[`response.finished`][] indicates whether the [`response.end()`] has been
```

Comment:
Should this mention what property replicas the existing behavior as well (I think you said .ended in the linked issue)?

Reply:
Pushed an update to try and further clarify.

Reply:
It still is unclear to me on how to keep the existing behavior. Which property replicates the same behavior as .finisher but is not deprectaed?

Reply:
Is it unclear in general or just based on the description?

Reply:
How about now?

Reply:
Yep, if `.writableFinished` == existing `.finished` behavior, it is clear to me now. I guess that `.ended` property from the issue was just a generic example, not the actual property name.

Reply:
No, `.writableEnded` == existing `.finished` behavior. This confusion is the exact reason why need to deprecate `.finished`.

Reply:
Haha, I even made the mistake myself. Pushed another update.